.gitignore

```diff
@@ -44,4 +33,7 @@ test/tests-working-dir
 dist/**
 IExec*/**
 utils/**
-errors/**
\ No newline at end of file
+errors/**
```

Comment:
why removing errors ? No file named errors seems to be removed ?

Reply:
looking at the whole diff this didn't change